src/components/molecules/input/ms-input.stories.ts

```diff
@@ -0,0 +1,36 @@
+import type { Meta, StoryObj } from '@storybook/web-components'
+import { html } from 'lit'
+
+import './ms-input'
+
+export default {
+  title: 'Input',
+  argTypes: {
+    value: { defaultValue: '' },
+    placeholder: { defaultValue: '' },
+    disabled: { options: [true, false], control: { type: 'select' } },
+    label: { defaultValue: 'Test' },
+    message: { defaultValue: 'Informative message' },
```

Comment:
I would think that this line is redundant with the nº 33. Please check if it's necessary to put this argType. 
For string manipulation, there's no need for control, as it will appear with only put the value as an `art` of the StoryObject

Reply:
done!